base/sysimg.jl

```diff
@@ -360,10 +360,10 @@ using .I18n
 
 # frontend
 include("initdefs.jl")
-include("Terminals.jl")
-include("LineEdit.jl")
-include("REPLCompletions.jl")
-include("REPL.jl")
+include("repl/Terminals.jl")
```

Comment:
This matches the style of all other includes in `sysimg.jl`, so I'm going to leave it. Normalizing path names to included files would probably make sense.